schema/api.yaml

```diff
@@ -0,0 +1,111 @@
+swagger: "2.0"
+
+info:
+  version: 1.0.0
+  title: bot-lax-adaptor api
+  description: a web interface to interacting with lax via the bot-lax-adaptor
+
+schemes:
+  - https
+
+#basePath: /
+
+definitions:
+  error:
+    type: object
+    properties:
+      status:
+        type: string
+        description: the final state of the article supplied
+      code:
+        type: string
+        description: a succinct identifier to help class the type of error
+      message:
+        type: string
+        description: a more verbose explanation of the problem
+      trace:
+        type: string
+        description: a very detailed block of text describing exactly where the problem is
+
+  success:
+    type: object
+    properties:
+      status:
+        type: string
+        description: the final state of the article supplied
+      ajson:
+        type: object
+        description: the final article-json, including overrides
+      override:
+        type: object
+        description: the overrides provided
+
+paths:
+  /xml:
```

Comment:
@thewilkybarkid can confirm paths and field names conform to our API style (this is not part of api-raml but it pays to be consistent)